python/paddle/v2/fluid/layers/nn.py

```diff
@@ -2177,3 +2178,130 @@ def transpose(x, perm, name=None):
         outputs={'Out': [out]},
         attrs={'axis': perm})
     return out
+
+
+def im2sequence(input,
+                filter_size=1,
+                stride=1,
+                padding=0,
+                name=None,
+                layer_attr=None):
```

Comment:
`param_attr`?

Reply:
Thx. Removed  layer_attr / param_attr because this hasn't parameter.